lib/etl/load/create_new_resource_record.rb

```diff
@@ -2,9 +2,9 @@ class CreateNewResourceRecord
   def write(attributes)
     record = Resource.create!(attributes)
 
-    ap "Title: #{record.title}"
-    ap 'Metadata: '
-    ap record.metadata
-    ap "Content: #{record.content.truncate(100)}"
+    ap "Title: #{record.title}" # rubocop:disable Rails/Output
```

Comment:
The idea is generally to catch your leftover `puts` statements, which is probably generally a good thing (I leave these in accidentally all the time).

Actually, even this `ap` line here is temporary and is used for debugging only, so it's truly a one-off exception.

I'm inclined to leave this as-is for now, but disable this cop immediately the next time it even smells like it's not helpful.